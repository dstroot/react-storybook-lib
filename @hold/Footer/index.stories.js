import React from "react";
import { storiesOf } from "@storybook/react";

// components
import Footer from "../Footer";

// Footer section
storiesOf("Footer", module)
  .addParameters({
    // default for all stories in this book
    info: {
      inline: true,
      header: false,
      text: `
        A simple view-only generic planet component.
      `
    }
  })
  .add("show Footer", () => <Footer />);
