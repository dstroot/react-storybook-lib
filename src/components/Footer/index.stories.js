import React from 'react';
import { storiesOf } from '@storybook/react';

// components
import Footer from '../Footer';

// Footer section
storiesOf('Footer', module).add('show Footer', () => <Footer />);
