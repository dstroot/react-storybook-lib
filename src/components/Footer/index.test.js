import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

describe('Footer', () => {
  it('it should render', () => {
    const component = renderer.create(<Footer />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
