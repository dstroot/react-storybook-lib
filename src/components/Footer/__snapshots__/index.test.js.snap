// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer it should render 1`] = `
<div
  className="container mt-5"
>
  <footer
    className="pt-4"
  >
    <div
      className="row"
    >
      <div
        className="col-12 col-md"
      >
        <a
          href="/"
        >
          <img
            alt="logo"
            className="mb-3"
            src="logo-horiz.svg"
          />
        </a>
      </div>
      <div
        className="col-6 col-md"
      >
        <h5>
          Features
        </h5>
        <ul
          className="list-unstyled text-small"
        >
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Cool stuff
            </a>
          </li>
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Random feature
            </a>
          </li>
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Team feature
            </a>
          </li>
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Stuff for developers
            </a>
          </li>
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Another one
            </a>
          </li>
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Last time
            </a>
          </li>
        </ul>
      </div>
      <div
        className="col-6 col-md"
      >
        <h5>
          Resources
        </h5>
        <ul
          className="list-unstyled text-small"
        >
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Resource
            </a>
          </li>
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Resource name
            </a>
          </li>
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Another resource
            </a>
          </li>
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Final resource
            </a>
          </li>
        </ul>
      </div>
      <div
        className="col-6 col-md"
      >
        <h5>
          About
        </h5>
        <ul
          className="list-unstyled text-small"
        >
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Team
            </a>
          </li>
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Locations
            </a>
          </li>
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Privacy
            </a>
          </li>
          <li>
            <a
              className="text-muted"
              href="/"
            >
              Terms
            </a>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-sm-8"
      >
        <p
          className="small text-muted"
        >
          Copyright Â© 
          2019
           Pacific Life Insurance Company, all rights reserved.
        </p>
      </div>
    </div>
  </footer>
</div>
`;
